main.js

```diff
@@ -28,6 +30,7 @@ module.exports = function(options) {
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
 		withRequestTracing: false,
+		hasHeadCss: false,
```

Comment:
i assume that we are currently requiring a one-to-one mapping between apps and head css's.  ie. an app can't have more than one compiled head css file?  (e.g. different routes)

(this is ok, keeps things simple, just checking)